packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -62,6 +63,44 @@ const propTypes = {
     ),
   ),
   /**
+   * An array of items to be displayed as help menu content.
+   */
+  helpItems: PropTypes.arrayOf(PropTypes.shape(
+    navElementShape,
+    {
+      children: PropTypes.arrayOf(PropTypes.shape(
+      navElementShape,
+        {
+          children: PropTypes.array,
```

Comment:
The children of children are ignored based on initial design discussion with Mike. We can discuss this issue of how many hierarchical levels we want to go. The help modal content list is now designed based on assumption that there will not be children of children and if that assumption is changed need to change that component also. For now keeping it as current form. If we decide not limiting children will refactor the code.